pkg/policy/api/cidr.go

```diff
@@ -50,6 +50,13 @@ type CIDRRule struct {
 	// +kubebuilder:validation:Optional
 	ExceptCIDRs []CIDR `json:"except,omitempty"`
 
+	// CIDRGroupRef is a reference to a CiliumCIDRGroup object.
```

Comment:
I thought to keep the field `ExceptCIDRs` close to `CIDRs` because logically tied to it, but not to `CIDRGroupRef`. At the moment, the translation does not support both `CIDRGroupRef` and `ExceptCIDRs`.
Probably it is better to move `ExceptCIDR` below but also add a comment to point out that specifying something there is not supported when using `CIDRGroupRef`.

Reply:
I documented it (I'm about to push a new revision of the code), but not enforced.
Regarding the enforcement, do you mean something at the kubebuilder validation level or something in the cidr_group handler? For the former I am not aware of anything like that in kubebuilder validation semantic (but I can take a deeper look), for the latter what should we do in that case? Maybe stop the translation and return an error? Or simply ignore the fromCIDRSet element and go ahead?

Reply:
Actually, I realize that support for `ExceptCIDRs` can be easily implemented simply copying the field to the translated rules. I added it alongside some unit testing.